.buildkite/pipeline.yml

```diff
@@ -1,7 +1,31 @@
----
+
+env:
+  DOCKER_REGISTRY_VAULT_PATH: secret/ci/elastic-cloud-on-k8s/docker-registry-elastic
+  DOCKER_REGISTRY_HOSTNAME: docker.elastic.co
+
 steps:
-  - label: "Test"
-    command: "true"
 
-  - label: "Build"
-    command: "true"
+  - label: "lint"
+    command: make lint
+    agents:
+      image: "golang:1.20.6"
+      cpu: "4"
+      memory: "8G"
+
+  - label: "test"
+    command: make unit-tests
+    agents:
+      image: "golang:1.20.6"
+      cpu: "4"
+      memory: "4G"
+
+  - label: "publish-image"
+    commands:
+      #- .buildkite/registry-login.sh
+      #- drivah build --push .
```

Comment:
Not needed? Or waiting for vault credentials?

Reply:
Waiting for creds.